plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -1732,12 +1734,6 @@ fu_synaptics_mst_device_setup(FuDevice *device, GError **error)
 	guid3 = g_strdup_printf("MST-%s", name_family);
 	fu_device_add_instance_id_full(FU_DEVICE(self), guid3, FU_DEVICE_INSTANCE_FLAG_QUIRKS);
 
-	/* this is not a valid customer ID */
-	if ((self->board_id >> 8) == 0x0) {
-		fu_device_inhibit(device,
-				  "invalid-customer-id",
-				  "cannot update as CustomerID is unset");
```

Comment:
If this is indeed a whitebox unit, then we need to set `FU_DEVICE_INTERNAL_FLAG_ENFORCE_REQUIRES` instead of the inhibit.

Reply:
Do you mean add this checking back but change `fu_device_inhibit()` to `return FU_DEVICE_INTERNAL_FLAG_ENFORCE_REQUIRES`?

Reply:
Exactly that.

Reply:
e.g. https://github.com/fwupd/fwupd/pull/6563

Reply:
since that's pulled, make sure you rebase the branch when you change this PR.